tests/utils/disable-download-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { disableDownload } from '../../src/utils/disable-download';
+import { isJest } from '../../src/utils/is-jest';
+
+jest.mock('../../src/utils/is-jest');
+
+describe('disableDownload', () => {
+  // versions below 11.0.0 are not available for download on darwin arm64
+  it('returns false for downloadable versions and true otherwise', () => {
+    if (process.platform == 'darwin' && process.arch == 'arm64') {
+      (isJest as any).mockReturnValueOnce(false).mockReturnValueOnce(false);
```

Comment:
This can just be `mockReturnValue(false)` which sets it to always return false.

It feels a bit strange to me to be mocking `isJest` to be false since the point of the helper is to detect that it's in Jest.

Reply:
I think I've come up with a better solution. I will make the changes recently. 😄